src/components/ScanPatientPage/BarcodeScan.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from "react";
+import { useHistory } from "react-router-dom";
+import Quagga from "quagga";
+import { useQuery } from "react-apollo";
+import { FETCH_ALL_PATIENTS } from "../../graphql/queries/patients";
+
+import "../../styles/BarcodeScan.css";
```

Comment:
should add a note for why raw css styling is used here (i believe you're looking into a way around this as well)